cmd/vclusterctl/cmd/platform/platform.go

```diff
@@ -4,8 +4,11 @@ import (
 	"github.com/loft-sh/vcluster/cmd/vclusterctl/cmd/platform/add"
 	"github.com/loft-sh/vcluster/cmd/vclusterctl/cmd/platform/connect"
 	"github.com/loft-sh/vcluster/cmd/vclusterctl/cmd/platform/get"
+	vimport "github.com/loft-sh/vcluster/cmd/vclusterctl/cmd/platform/import"
```

Comment:
no need to make this an alias

Reply:
Same as [here](https://github.com/loft-sh/vcluster/pull/1803#discussion_r1618801894). Due to import being a reserved keyword.